docs/content/preview/explore/ysql-language-features/postgresql-compatibility.md

```diff
@@ -16,10 +16,95 @@ YugabyteDB is a [PostgreSQL-compatible](https://www.yugabyte.com/tech/postgres-c
+
+- Performance parity
+
+    Performance parity refers to the capabilities of YugabyteDB that allow applications running on PostgreSQL to run with predictable performance on YugabyteDB. In other words, the performance degradation experienced by small and medium scale applications going from a single server database to a distributed database should be predictable and bounded.
+
+## Enhanced Postgres Compatibility Mode
+
+To test and take advantage of features developed for PostgreSQL compatibility in YugabyteDB that are currently in {{<badge/ea>}}, you can enable Enhanced Postgres Compatibility Mode. When this mode is turned on, YugabyteDB is configured to use all the latest features developed for feature and performance parity. Enhanced Postgres Compatibility Mode is available in v2024.1 and later.
+
+Depending on the version of YugabyteDB, Enhanced Postgres Compatibility Mode configures a different set of features as described in the following sections.
+
+After turning this mode on, as you upgrade universes, YugabyteDB will automatically enable new designated PostgreSQL compatibility features.
+
+As features included in the PostgreSQL compatibility mode transition from {{<badge/ea>}} to {{<badge/ga>}} in subsequent versions of YugabyteDB, they become enabled by default on new universes, and are no longer managed under Enhanced Postgres Compatibility Mode on your existing universes after the upgrade.
+
+{{<note title="Note">}}
+If you have set these features independent of Enhanced Postgres Compatibility Mode, you cannot use Enhanced Postgres Compatibility Mode.
+
+Conversely, if you are using Enhanced Postgres Compatibility Mode on a universe, you cannot set any of the features independently.
+{{</note>}}
+
+### v2024.1
+
+| EA Feature | Flag |
+| :--- | :--- |
+| Read-Committed isolation | yb_enable_read_committed_isolation=true |
+| Wait-on-Conflict concurrency for predictable P99 latencies | enable_wait_queues=true |
+| Cost based optimizer. Includes query pushdowns, LSM indexes, and batched nested loop joins for PostgreSQL-like performance. | yb_enable_base_scans_cost_model=true<br>yb_bnl_batch_size=1024<br>yb_fetch_row_limit=0<br>yb_fetch_size_limit=1MB |
+| Use range sharding (ascending) by default | yb_use_hash_splitting_by_default=false |
+
+### v2024.2
+
+The following shows the features that are planned for Enhanced Postgres Compatibility Mode for v2024.2 and is subject to change without notice.
+
+| EA Feature | Flag |
+| :--- | :--- | :--- |
+| Feature 1 | flag=false |
+| Feature 2 | flag=true |
+| Feature 3 | flag=false |
+
+The following features are planned to go {{<badge/ga>}} in v2024.2 and will no longer be managed under Enhanced Postgres Compatibility Mode:
+
+- Read-Committed isolation
```

Comment:
ReadCommited and range sharding are TBD to be turned on by default. CC @mihnea

Reply:
@sushantrmishra @m-iancu Haven't had any movement on this, any chance we can get it over the line at some point